apps/prairielearn/elements/pl-file-editor/pl-file-editor.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-env browser,jquery */
-/* global ace, showdown, MathJax, DOMPurify */
+/* global ace, showdown, MathJax, filterXSS */
```

Comment:
Make sure you start with the current version of the element. This change is undoing recent changes to this element.

Reply:
The latest commit pulled the recent changes, so this should be fixed now. Thanks!